server/src/sim/makeworkbook.py

```diff
@@ -602,29 +602,46 @@ def create(self, path):
         self.book.close()
 
 class OptimaGraphTable:
-    def __init__ (self, name, columns, verbose = 2):
+    def __init__ (self, name, sheets, verbose = 2):
         self.name = name
-        self.columns = columns
         self.verbose = verbose
+        self.sheets = sheets
 
     def create(self, path):
```

Comment:
I think this is presuming we may eventually have more methods in this class. If we implement this in the constructor, we will not have the opportunity to expand later.